### ISSUE TITLE
do you have try this code on voc07+12?

### ISSUE THREAD

**Issue by leishi2018** [Kongsea/FPN_TensorFlow#2]
I try  this code on voc07/12. But it is not good at find other object but person.The car dog car can't be found in the picture.only a few car or other object can be found.

**Comment by Kongsea** [comment]
Sorry, I have not tested it.
Maybe you can have a try.
Besides, I always use Detectron now. Maybe you can have a try to switch to it too.

**Comment by zhanhuanli** [comment]
@leishi2018 @Kongsea  我想请问如何在VOC2012数据集上运行起代码？比如dataset_name=‘person’，添加标签集合为：elif cfgs.DATASET_NAME == 'person':
    NAME_LABEL_MAP = {
        'back_ground': 0,
        "person": 1,
        'aeroplane': 0,
        'bicycle': 0,
        'bird': 0,
        'boat': 0,
        'bottle': 0,
        'bus': 0,
        'car': 0,
        'cat': 0,
        'chair': 0,
        'cow': 0,
        'diningtable': 0,
        'dog': 0,
        'horse': 0,
        'motorbike': 0,
        'pottedplant': 0,
        'sheep': 0,
        'sofa': 0,
        'train': 0,
        'tvmonitor': 0
    }
，调整训练参数，可是依旧一直报InvalidArgumentError (see above for traceback): LossTensor is inf or nan : Tensor had NaN values
	 [[Node: train_op/CheckNumerics = CheckNumerics[T=DT_FLOAT, message="LossTensor is inf or nan", _device="/job:localhost/replica:0/task:0/device:GPU:0"](control_dependency)]]
	 [[Node: Momentum/update_resnet_v1_101/block3/unit_17/bottleneck_v1/conv3/BatchNorm/beta/ApplyMomentum/_2540 = _Recv[client_terminated=false, recv_device="/job:localhost/replica:0/task:0/device:CPU:0", send_device="/job:localhost/replica:0/task:0/device:GPU:0", send_device_incarnation=1, tensor_name="edge_13388_Momentum/update_resnet_v1_101/block3/unit_17/bottleneck_v1/conv3/BatchNorm/beta/ApplyMomentum", tensor_type=DT_FLOAT, _device="/job:localhost/replica:0/task:0/device:CPU:0"]()]]
错误，我想我是哪里出来问题，如果你们可以告诉我，万分感谢~

**Comment by Kongsea** [comment]
Please refer to [this](https://github.com/Kongsea/FPN_TensorFlow#errors-may-encountered).
Or maybe it's because your learning rate is too large.

**Comment by zhanhuanli** [comment]
VOC上的四个坐标会不满足关系吗？ 我觉得问题可能不出在这个地方， 我的学习率是0.00005，还会太大吗，其余参数如你一样。

**Comment by zhanhuanli** [comment]
@Kongsea  I set lr to a very small value, and also changed the data to tfrecord. I also changed the data set to 20 types - pascal tried, or the above problem occurs. Do you know why this is the case? ? Have you encountered similarities when using VOC datasets? Where am I wrong?

**Comment by Kongsea** [comment]
我猜想可能是因为你修改了数据集的label_map，是否会导致class、label还有数据这些地方的数目无法匹配了？

**Comment by zhanhuanli** [comment]
我是增加了label_map的person，如上贴图所示，这样的形式只有person一个类别，其余都属于背景，这样class和label数据之间应该都能匹配。我也尝试了直接用原本的pascal ，类别设为20类，也就是voc2012的类别数，这应该就不会存在这样的问题，可还是出现这类问题。你是直接拿自己的数据拿去运行的吗？还是你也尝试过coco或者voc

**Comment by Kongsea** [comment]
我是直接在自己的数据集上跑的，没有试过COCO或者voc。
而且我也没有试过你这种类别的形式，我是把所有类别都算作有效类别了。
我怀疑你的问题应该就出在这里。你可以试一下看看。

**Comment by zhanhuanli** [comment]
对的， 我考虑过这个问题 ，我也把所有类别（20类）都当成有效类别测试了，还是如此。 你的数据集私密吗？ 方便共享一份吗？

**Comment by Kongsea** [comment]
不好意思，因为是公司项目，所以数据不便共享，请多见谅。
另，如果你使用FPN的话，建议尝试Detectron，个人感觉那个更稳定可靠一些。

**Comment by zhanhuanli** [comment]
好的，没事~ 我也只是做个测验，行，我有空试试，麻烦了~
